scripts/cell_simulation.py

```diff
@@ -164,8 +179,8 @@ def gen_sim_data(
     ##############################################################################
     # Generate Trees
     ##############################################################################
-    #  generate random trees (uniform sampling) as adjacency matrix
-    tree = tree_inf.generate_random_tree(rng_tree, n_nodes=n_mutations)
+    #  generate random trees (uniform sampling) as adjacency matrix / add +1 for root
+    tree = tree_inf.generate_random_tree(rng_tree, n_nodes=n_mutations + 1)
```

Comment:
Found a bug! - the `cell_simulation.py` did generate random trees with one mutation to little.

Reply:
Right, good you found it!